utils/to-doc.js

```diff
@@ -1,12 +1,16 @@
 'use strict'
 
 var extend = require('pouchdb-extend')
+var now = require('./now')
 
 module.exports = function objectToDoc (object) {
   var doc = extend({}, object, {
     _id: object.id
   })
 
+  doc.updatedAt = now()
+  doc.createdAt = doc.createdAt || doc.updatedAt
```

Comment:
Ah okay, `toDoc` is actually the other way, taking a CouchDB `doc` and turning it into a hoodie-esque `object`, so [utils/to-object.js](utils/to-object.js) might be the better place to put this